src/In.ProjectEKA.HipService/DataFlow/DataFlowArtefactRepository.cs

```diff
@@ -0,0 +1,23 @@
+
+namespace In.ProjectEKA.HipService.DataFlow
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using In.ProjectEKA.HipLibrary.Patient.Model;
+    
+    public class DataFlowArtefactRepository : IDataFlowArtefactRepository
+    {
+        public async Task<Tuple<DataFlowArtefact, ErrorRepresentation>> GetFor(
+            HealthInformationRequest healthInformationRequest)
+        {
+            var dataFlowMessage = new DataFlowArtefact(
+                new List<GrantedContext>{new GrantedContext("5",
```

Comment:
if you don't have an I/O call, for now you can just remove async.

Reply:
Done